### ISSUE TITLE
Ubuntu install directives wrong

### ISSUE THREAD

**Issue by chmike** [visualfc/liteide#679]
Hello the Ubuntu directives to build the package from sources are wrong.

The following lines
```
$ cd liteide/build
$ ./update_pkg.sh
$ QTDIR=/usr ./build_linux.sh
```
must be replaced by

```
$ cd liteide/build
$ ./update_pkg.sh
$ export QTDIR="/usr"
$ ./build_linux.sh
```

**Comment by nkts** [comment]
I does not really make any difference - $QTDIR will be passed to build_linux.sh in both cases. Unless you depend on it after this command.

**Comment by vovkasm** [comment]
Yes, @nkts is right. Most probably requestor uses incorrect wording... not "wrong", but not coherent with rules for others (Linux, Mac OS X, OpenBSD). All others *nix variants uses `export QTDIR` because QTDIR required by more than one script file ;-)
